src/libs/DistanceRequestUtils.ts

```diff
@@ -78,6 +110,39 @@ function getRoundedDistanceInUnits(distanceInMeters: number, unit: Unit): string
     return convertedDistance.toFixed(2);
 }
 
+/**
+ * @param hasRoute Whether the route exists for the distance request
+ * @param unit Unit that should be used to display the distance
+ * @param rate Expensable amount allowed per unit
+ * @param currency The currency associated with the rate
+ * @param translate Translate function
+ * @param toLocaleDigit Function to convert to localized digit
+ * @returns A string that describes the distance traveled and the rate used for expense calculation
+ */
+function getRateForDisplay(
+    hasRoute: boolean,
+    unit: Unit | undefined,
+    rate: number | undefined,
+    currency: string | undefined,
+    translate: LocaleContextProps['translate'],
+    toLocaleDigit: LocaleContextProps['toLocaleDigit'],
+    isOffline?: boolean,
+): string {
+    if (isOffline && !rate) {
+        return translate('iou.defaultRate');
+    }
+    if (!hasRoute || !rate || !currency || !unit) {
+        return translate('iou.routePending');
```

Comment:
> I think we agreed on just showing `Pending` for rate.
> 
fixed

> @koko57 and if we have the rate, why do we need the route to show the rate?

 @neil-marcellini should we show the rate in such case?

Reply:
> @koko57 and if we have the rate, why do we need the route to show the rate?

Yes as I said before, `hasRoute` should not be a consideration for showing the rate, since it has nothing to do with getting a rate.